scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/update_submission/"
 submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN'"
```

Comment:
The above API expects submission pk to be in request data. Please add that change

Reply:
@Ram81, Sure,
Added it in the code, also tested it locally by creating a challenge and submitting using this script, also tested the formation of new message.